assets/section-featured-blog.css

```diff
@@ -7,8 +7,12 @@
 }
 
 @media screen and (min-width: 750px) {
+  .blog:not(.background-secondary) {
+    margin: calc(5rem + var(--page-width-margin)) 0;
+  }
+
   .blog.background-secondary {
-    padding-bottom: 5rem;
+    padding-bottom: calc(5rem + var(--page-width-margin));
```

Comment:
@ludoboludo The spacing is different for this section on top and bottom even without the title. I just checked `main` and this is expected which I think makes sense because you dont need a lot of spacing since there is no title/button

Reply:
I think it might have been a miss from before 🤔  It looks better on the Multi-column section imo: ![](https://screenshot.click/21-09-xilez-aygcs.png)

I will loop in @Oliviammarcello to see what she thinks :)

Reply:
The spacing should be the same! The space on the top of the blog posts should match what is below the "view all" button